\subsection*{Metaproteome analysis and quantitative PCR}

From each harvest (14, 97, 181, and 475 days), one replicate per litter type was stored at -80\textdegree C for metaproteome analysis. 3 g of each sample were grounded in liquid nitrogen and extracted with Tris/KOH buffer (pH 7.0) containing 1\% SDS. Samples were sonicated for 2 min, boiled for 20 min and shaken at 4 \textdegree C for 1 h. Extracts were centrifuged twice to remove debris and concentrated by vacuum-centrifugation. An aliquot of the sample was applied to a 1D-SDS-PAGE and subjected to in-gel tryptic digestion. The resulting peptide mixtures were analyzed on a hybrid LTQ-Orbitrap MS (Thermo Fisher Scientific) as described earlier \cite{Schneider2010}. Protein database search against the UniRef 100 database, which also comprised the translated metagenome of the microbial community of a Mennesota farm silage soil \cite{Tringe2005} and known contaminants, was performed using the MASCOT Search Engine. A detailed description of the extraction procedure and search criteria was published by \cite{Keiblinger2011}. If more than one protein was identified based on the same set of spectra these  proteins were grouped together resulting in one protein cluster. The obtained protein/protein cluster hits were assigned to phylogenetic and functional groups and assignments were validated by the PROPHANE workflow (http://prophane.svn.sourceforge.net/viewvc/prophane/trunk/; \cite{Schneider2011}). Higher protein abundance is represented by a higher number of MS/MS spectra acquired from peptides of the respective protein. Thus, protein abundances were calculated based on the normalised spectral abundance factor (NSAF) \cite{Florens2006, Zybailov2006}. This number allows relative comparison of protein abundances over different samples \cite{Bantscheff2007}. Protein abundances was aggregated at class level for fungi and protebacteria and at phylum level for other bacterial taxa. These abundances were subjected to a canonical correspondance analysis without constrains. Vectorial fittings of stoichiometrical ratios (litter, microbial biomass and imbalance) were calculated and plotted when p \textless 0.05.

Quantitative PCR was used to determine fungal and bacterial abundance as described recently \cite{Inselsbacher2010}. F:B ratios were calculated as the ratio between estimated amounts of bacterial and fungal DNA found.